.circleci/config.yml

```diff
@@ -9,6 +9,11 @@ executors:
     docker:
       - image: circleci/node:12.7
     working_directory: ~/repo
+    environment:
+      GIT_AUTHOR_EMAIL: tom.vycital+vis-bot@gmail.com
```

Comment:
:confused: Wouldn't it be better to have a generic email Adresse without your name in it. This way we could give it also tho other maintainers!?

Reply:
Yeah, we can change it at any time.